src/obr/cli.py

```diff
@@ -292,6 +308,148 @@ def query(ctx: click.Context, **kwargs):
     query_impl(project, queries, output=True, latest_only=not kwargs.get("all"))
 
 
+@cli.command()
+@click.option(
+    "--filter",
+    type=str,
+    multiple=True,
+    help=(
+        "Pass a <key>=<value> value pair per occurrence of --filter. For instance, obr"
+        " archive --filter solver=pisoFoam --filter preconditioner=IC"
+    ),
+)
+@click.option(
+    "-f",
+    "--folder",
+    required=True,
+    default=".",
+    type=str,
+    help="Path to OpenFOAMProject.",
+)
+@click.option(
+    "-r",
+    "--repo",
+    required=True,
+    type=str,
+    help=(
+        "Path to data repository. If this is a valid Github repository, files will be"
+        " automatically added."
+    ),
+)
+@click.option(
+    "-s",
+    "--skip-logs",
+    required=False,
+    is_flag=True,
+    help=(
+        "If set, .log files will not be archived. This does not affect .log files"
+        " passed via the --file option."
+    ),
+)
+@click.option(
+    "-a",
+    "--file",
+    required=False,
+    multiple=True,
+    help="Path(s) to non-logfile(s) to be also added to the repository.",
+)
```

Comment:
I would suggest to add an option --tag, to "tag" archival cases. So one can use for example `obr archive --tag HoReKa` to write archived files to `<target>/workspace/<job.id>/HoReKa/`  instead of `<target>/workspace/<job.id>/case/`. We would need to adapt `obr run -o apply` to work with files in the tag folder instead of case but that shouldn't be a problem

Reply:
Good idea, I will also add a `--dry-run` option to enable a verbose test run.

Reply:
`--dry-run` would be still a good option, since with `--skip-logs` active we would still copy signac files

Reply:
Good point